modules/local/filter_blastn.nf

```diff
@@ -10,6 +10,7 @@ process FILTER_BLASTN {
     input:
     tuple val(meta), path(hits)
     path header
+    path filtered_header
 
     output:
     tuple val(meta), path('*.txt'), emit: txt
```

Comment:
```suggestion
    tuple val(meta), path('*filter.blastn.txt'), emit: txt
```

Maybe modify to this then to not have a [meta, [*.blastn.txt, *.filter.blastn.txt]]?

Reply:
Thank you! I didn't see this one!